lib/telephony/alert_sender.rb

```diff
@@ -41,26 +41,26 @@ def send_personal_key_sign_in_notice(to:, country_code:)
       response
     end
 
-    def send_join_keyword_response(to:, country_code:)
-      message = I18n.t('telephony.join_keyword_response')
-      response = adapter.send(message: message, to: to, country_code: country_code)
-      log_response(response, context: __method__.to_s.gsub(/^send_/, ''))
-      response
-    end
+    # def send_join_keyword_response(to:, country_code:)
+    #   message = I18n.t('telephony.join_keyword_response')
```

Comment:
now we can delete the corresponding translation strings, right? Like `telephony.join_keyword_response`